MediaBrowser.Model/Querying/QueryResult.cs

```diff
@@ -17,6 +17,12 @@ public class QueryResult<T>
         /// <value>The total record count.</value>
         public int TotalRecordCount { get; set; }
 
+        /// <summary>
+        /// The index of the first record in Items
+        /// </summary>
+        /// <value>First record index</value>
```

Comment:
I believe these need to have periods at the end, but that's the only thing I see wrong with this pull request.

Reply:
Ok, added those.